CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## TBD
+### Changed
+* Default Housekeeping are now provided vi `@Bean` `housekeepingEnvironment` only if bean if not provided.
```

Comment:
vi? or emacs? ;p

Reply:
Maybe say:

"Default Housekeeping configuration is now provided via @Bean` `housekeepingEnvironment` only if the bean is not already provided."

Reply:
yeah, missed few words